Rakefile

```diff
@@ -16,6 +18,39 @@ Rake::TestTask.new(:test) do |t|
   t.verbose = false
 end
 
+namespace :test do
+  desc "Run tests for templates"
+  task :templates => ["templates:all"]
+
+  namespace :templates do
+    task :all => [:npm, :rackup, :mocha, :kill]
+
+    work_dir    = Pathname(__FILE__).dirname + "test/templates"
+    pid_file    = Pathname(Dir.tmpdir) + "web_console.#{SecureRandom.uuid}.pid"
+    server_port = 29292
+    rackup_opts = "-D -P #{pid_file} -p #{server_port}"
+
+    task :npm do
+      Dir.chdir work_dir
+      system "npm install --silent"
```

Comment:
`Dir.chdir(work_dir) { system "npm install --silent" }` so you preserve the current CWD.